agent/format-2.0.go

```diff
@@ -67,6 +67,7 @@ type format_2_0Serialization struct {
 	QueryTracingEnabled   bool          `yaml:"querytracingenabled,omitempty"`
 	QueryTracingThreshold time.Duration `yaml:"querytracingthreshold,omitempty"`
 	DqlitePort            int           `yaml:"dqlite-port,omitempty"`
+	SSHServerHostKey      string        `yaml:"ssh-server-host-key,omitempty"`
```

Comment:
could we rename it to `SSHHostKey`?

Reply:
I think as it maps directly to the bootstrap params key, it makes the code more traceable keeping it consistent?

Reply:
Consistency is key here, I agree with @ale8k